RefillStation/RefillStation/Presentation/OnboardingScene/TermsDetailViewController.swift

```diff
@@ -52,6 +53,13 @@ final class TermsDetailViewController: UIViewController {
         }
     }
 
+    private func moveToTop() {
+        let contentHeight = termsTextView.contentSize.height
+        let offSet = termsTextView.contentOffset.x
+        let contentOffset = contentHeight - offSet
+        termsTextView.contentOffset = CGPoint(x: 0, y: -contentOffset)
```

Comment:
```suggestion
        termsTextView.setContentOffset(.zero, animated: false)
```

이렇게 해도 같은 동작할 것 같습니다!

Reply:
@Neph3779  해당 코드로 설정시에도 중간 부분부터 시작하게됩니다....🥲
|![IMG_3342](https://user-images.githubusercontent.com/70168249/218394364-13c7fbd3-7508-49b0-971f-8ef045ee0780.PNG) | ![IMG_3341](https://user-images.githubusercontent.com/70168249/218394605-cd29367c-2040-4ea3-902c-8537d4c32d2c.PNG) |
| :----------------------------------------------------------: | :----------------------------------------------------------: |
|                             문제화면                            |                             정상화면                             |

Reply:
앗 그렇군요!... 확인했습니다!